R/accordion.R

```diff
@@ -18,7 +21,7 @@
 #'
 #' @export
 #' @seealso [accordion_panel_set()]
-#' @examplesIf interactive()
+#' @examplesIf rlang::is_interactive()
```

Comment:
`rlang::is_interactive()` is better because it follows the global `rlang_interactive` option. That way we can set `options(rlang_interactive = FALSE)` and render the pkgdown site in an interactive console without having the app examples halt the rendering process.